package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "honeyswap-default-token-list",
-  "version": "1.3.2",
+  "version": "2.0.2",
```

Comment:
Let's leave this at 2.0.0 - the GitHub action will bump it automatically to 2.0.1.